x-pack/plugins/cases/server/saved_object_types/cases.ts

```diff
@@ -52,9 +52,6 @@ export const caseSavedObjectType: SavedObjectsType = {
       },
       connector: {
         properties: {
-          id: {
```

Comment:
These are removed from the saved object type and stored in the saved object `references` array now.